sdk/core/azure-core/src/http/logging_policy.cpp

```diff
@@ -7,7 +7,7 @@
 #include <chrono>
 #include <sstream>
 
-using namespace Azure::Core;
+using Azure::Core::Context;
```

Comment:
@CaseyCarter - the guidelines don't mention anything about this, so I am wondering, is it fine to use using statement declarations like this?